apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -352,6 +363,12 @@ const REDUX_TRACKING = {
 	'core/notices': {
 		createErrorNotice: trackErrorNotices,
 	},
+	'core/edit-site': {
+		setIsListViewOpened: trackListViewToggle,
+	},
+	'core/edit-post': {
```

Comment:
I wonder given that the new Widgets screen will likely land in WP 5.8, whether you should add `edit-widgets` to this list for tracking (obviously check it fails silently as it's not in Core yet)?

<img width="1170" alt="Screen Shot 2021-06-09 at 09 50 48" src="https://user-images.githubusercontent.com/444434/121324122-57822380-c908-11eb-86cf-5829372e5d7c.png">

Reply:
`edit-widgets` is still using the old dropdown list view. Which isn't tracked in the redux store.